src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/CreateVacancyReviewCommandHandler.cs

```diff
@@ -55,6 +55,14 @@ public async Task Handle(CreateVacancyReviewCommand message, CancellationToken c
             var vacancy = vacancyTask.Result;
             var previousReviews = previousReviewsTask.Result;
 
+            //Defensive code, just in case the message is republished due to an error after creating the review. 
+            var activePreviousReview = previousReviews.FirstOrDefault(r => r.Status != ReviewStatus.Closed);
```

Comment:
Could use .Any()

Reply:
I am logging the id of the active review below, hence needed a reference to it.